src/xbps/repositories/official/installing.md

```diff
@@ -1,12 +1,8 @@
 # Installing Repositories
 
 Void provides official [repositories](./index.md) that may be installed via a
-package. For example, to install the `nonfree` repository, install the package
```

Comment:
Suggest "that may be enabled via the installation of the relevant package. For example, to enable the".

Reply:
Good point. Fixed.